src/components/profile/ProfileService.js

```diff
@@ -165,7 +165,7 @@ goog.require('ga_urlutils_service');
           $http(params).success(function(data, status) {
             // When all the geometry is outside switzerland
             if (data.length == 0) {
-              data = [{alts: {COMB: 0}, dist: 0}];
+              data = [{alts: {}, dist: 0}];
```

Comment:
I know it was like that before and it's minor just a useless duplicate piece of code